core/coreunix/add.go

```diff
@@ -424,6 +439,29 @@ func (adder *Adder) addDir(path string, dir files.Directory, toplevel bool) erro
 
 	return it.Err()
 }
+const DEBUG = false
+func (adder *Adder) setTokenMetadata(m *map[string]interface{}) {
```

Comment:
If you are already working on the `btfs add` option, maybe you can just close this PR and do this inside that one?

Reply:
Yes, agreed. I will do that way.